lms/djangoapps/teams/static/teams/templates/edit-team.underscore

```diff
@@ -0,0 +1,21 @@
+<div class="wrapper-msg is-incontext urgency-low warning">
+    <div class="msg">
+        <div class="msg-content">
+            <h3 class="title"><%= gettext("Oops!") %></h3>
```

Comment:
After huddling, the Doc team members agreed that we'd prefer not to use "Oops!" for tone as well as difficulties with translation. @frrrances let's go with no title here. Thanks!